nectar/src/order.rs

```diff
@@ -66,7 +66,7 @@ impl BtcDaiOrderForm {
 
         Ok(BtcDaiOrderForm {
             position: Position::Sell,
-            quantity: base_amount.into(),
+            quantity: Quantity::new(base_amount),
```

Comment:
Is it a sign that `From` implementation should be done?

Reply:
I am not sure to be honest.

I am not abstracting over the construction of `Quantity` anywhere so I don't see anything wrong with a using an explicit constructor. For a type like `Quantity` that enforces invariants, I actually prefer it being explicit so it stands out among all the other `.into()` calls. Can't really present any better argument though :)

Reply:
I'm with @D4nte on this one, although I don't feel strongly, I prefer the from/into version than new/to_inner.